app/services/bucket_service.rb

```diff
@@ -8,9 +9,11 @@ def self.send_bucket_created_emails(bucket: )
   end
 
   def self.send_bucket_live_emails(bucket: )
-    memberships = bucket.group.memberships.active.reject { |membership| membership.member == bucket.user }
-    memberships.each do |membership|
-      member = membership.member
+    group = bucket.group
+    members = bucket.participants(exclude_author: true, subscribed: true)
+    members.each do |member|
```

Comment:
@euglazer I think you should sort our archived members with a scope, and just iterate over those : 

```ruby 
members.published.each do |member|
```

if you think thats a bad idea, or are unclear how to execute please ask Rob / me